src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -1176,7 +1176,7 @@ default MessageEditAction editMessageComponents(@Nonnull LayoutComponent... comp
      *
      *     <li>{@link net.dv8tion.jda.api.requests.ErrorResponse#UNKNOWN_MESSAGE UNKNOWN_MESSAGE}
      *     <br>The provided {@code messageId} is unknown in this MessageChannel, either due to the id being invalid, or
-     *         the message it referred to has already been deleted. This might also be triggered for ephemeral messages.</li>
```

Comment:
This is still true if the interaction expired I believe

Reply:
for all methods or just the `editMessageAttachments` methods?

Reply:
All of them